modules/ROOT/pages/troubleshooting.adoc

```diff
@@ -364,16 +364,17 @@ and causes an unexpected error, such as CWWKS9104A.
 
 === Application installation causes unexpected security behavior
 
-If you specify your application in both the `server.xml` files by using the application element and in the dropins folder, the application installation is attempted twice and the security configuration in the `server.xml` file might or might
+If you specify your application in the server.xml file by using both the application element and in the dropins folder, the application installation is attempted twice and the security configuration in the `server.xml` file might or might
 not take effect.
 
-You might see the following message,
+You might see the following message:
 
 ----
-CWWKZ0013E: It is not possible to start two applications called {0} followed by unexpected
-security behavior and error messages such as CWWKS9104A.
+CWWKZ0013E: It is not possible to start two applications called {0}. 
 ----
 
```

Comment:
so as not to lose the info that was removed for the message block:

This message might be followed by unexpected security behavior and error messages such as CWWKS9104A.